broadinstitute/pearl
# JN-265 updating admin login ui

The prior admin login was confusing as to what the default action should be.  This makes it clearer that the b2c login path is the default, and also adds some Juniper theming.  Erin gave the new look her blessing as being a substantial upgrade.

![image](https://user-images.githubusercontent.com/2800795/230625955-ce7bacde-69ce-454a-8309-80265619587c.png)

TO TEST:
1. go to admin tool
2. confirm login and development mode login still work
